app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -470,9 +478,6 @@ class MainActivity : AppCompatActivity() {
 
     private fun setSignClassificationMode() {
         soundsPlayer.stop()
-        appModelPerformanceConfig = ModelPerformanceConfig.Merged(
```

Comment:
if we don't change `appModelPerformanceConfig ` so we don't need recall `VisionManager.setModelPerformanceConfig(xxx)`

Reply:
Will fix